packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-# Unreleased (0.8.0)
+# Unreleased
+
+# 0.8.2
+- [fixed] Fixed an issue where native ES6 module loading was not working.
+
+# 0.8.1
+- [fixed] Fixed an issue where typings are created in the wrong location.
```

Comment:
Is there a way to rephrase this somehow to represent a user-visible difference in the SDK?  This seems to be describing the cause of the issue rather than the symptom.  Also "are" should be "were" to match the tense of "fixed"

Reply:
Leaving as is since this matches the public release notes.